tests/Editor/EditorTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Tests\Unit\Hooks;
+
+use Brain\Monkey;
+use Brain\Monkey\Filters;
+use EightshiftForms\Editor\Editor;
+
+use function Tests\setupMocks;
+
+/**
+ * Mock before tests.
+ */
+beforeEach(function () {
+	Monkey\setUp();
+	setupMocks();
+	$this->editor = new Editor();
+});
+
+afterEach(function() {
+	Monkey\tearDown();
```

Comment:
One thing @iruzevic mentioned to me when I refactored the libs tests is to cleanup the `$this->editor` property.

Reply:
Will do 😄